website/src/main/tut/docs/rules/NoFinalize.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+title: NoFinalize
+---
+
+** syntactic **
+
+# NoFinalize
+
+Disallow implementing java.lang.Object.finalize.
+
+More info about finalize on this [blog](https://dzone.com/articles/javas-finalizer-is-still-there)
+
+_Since 0.5.6_
+
+...
+
+Example:
+
+```scala
+MyCode.scala:7: error: [NoFinalize]
```

Comment:
Missing message?

Reply:
Maybe we should extract this from the test suite?